lib/cookie.js

```diff
@@ -807,7 +807,7 @@ const cookieDefaults = {
   pathIsDefault: null,
   creation: null,
   lastAccessed: null,
-  sameSite: "none"
+  sameSite: undefined
```

Comment:
yeah, i'm torn on this detail.  we could change it to `null` to be more consistent but `undefined` seems to convey the intent better.

Reply:
As we discussed in our meeting; I think `undefined` is a good choice. This is a bit hard to do, since we really built this as part of an earlier draft. But inline with that; I think `undefined` works well.